test/integration/testPutGet.js

```diff
@@ -657,3 +658,329 @@ describe('PUT GET test with GCS_USE_DOWNSCOPED_CREDENTIAL', function ()
     );
   });
 });
+
+describe('PUT GET test multiple files', function ()
+{
+  this.timeout(100000);
+
+  var connection;
+
+  before(function (done)
+  {
+    connection = testUtil.createConnection();
+    connection.gcsUseDownscopedCredential = true;
+    testUtil.connect(connection, done);
+  });
+
+  after(function (done)
+  {
+    testUtil.destroyConnection(connection, done);
+  });
+
+  it('testDownloadMultifiles', function (done)
+  {
+    var tmpFile1, tmpFile2;
+    var createTable = `create or replace table ${TEMP_TABLE_NAME} (${COL1} STRING, ${COL2} STRING, ${COL3} STRING)`;
+    var removeFile = `REMOVE @${DATABASE_NAME}.${SCHEMA_NAME}.%${TEMP_TABLE_NAME}`;
+    var dropTable = `DROP TABLE IF EXISTS ${TEMP_TABLE_NAME}`;
+
+    // Create two temp file with specified file extension
+    tmpFile1 = tmp.fileSync({ postfix: 'gz' });
+    tmpFile2 = tmp.fileSync({ postfix: 'gz' });
+    // Write row data to temp file
+    fs.writeFileSync(tmpFile1.name, ROW_DATA);
+    fs.writeFileSync(tmpFile2.name, ROW_DATA);
+
+    var putQuery1 = `PUT file://${tmpFile1.name} @${DATABASE_NAME}.${SCHEMA_NAME}.%${TEMP_TABLE_NAME}`;
+    // Windows user contains a '~' in the path which causes an error
+    if (process.platform == "win32")
+    {
+      var fileName = tmpFile1.name.substring(tmpFile1.name.lastIndexOf('\\'));
+      putQuery1 = `PUT file://${process.env.USERPROFILE}\\AppData\\Local\\Temp\\${fileName} @${DATABASE_NAME}.${SCHEMA_NAME}.%${TEMP_TABLE_NAME}`;
+    }
+
+    var putQuery2 = `PUT file://${tmpFile2.name} @${DATABASE_NAME}.${SCHEMA_NAME}.%${TEMP_TABLE_NAME}`;
```

Comment:
can we create local putFileQuery method to generate proper queries with win32 specific behaviour written and commented in one place?